scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -441,7 +441,7 @@ <h6><span class="fa fa-medkit ms-3 mt-2"></span>
                 <div class="col-8">
                   {% for omim in variant_phenotypes.omim_terms %}
                     <span class="badge badge-sm bg-secondary">
-                      <a class="text-white" target="_blank" href="http://omim.org/entry/{{omim|replace('OMIM:','')}}">
+                      <a target="_blank" href="http://omim.org/entry/{{omim|replace('OMIM:','')}}">
```

Comment:
One could also consider adding back a card header background - I sort of ignored that one, as it was one more color to choose. But this works!